topics/metabolomics/tutorials/lcms-preprocessing/slides.html

```diff
@@ -74,43 +74,43 @@
 
 ## xcms - groupChromPeaks
 
-![Large set of tables, in three groups, pool 181, 182 and 183. The top group is labelled independent peaklists. The next row of tables is group ions by m/z, and the rows in the table are coloured and lined up by their colours. The third group is labelled group by retention time, and the coloured rows are now merged into one big table, each colour on it's own row. The resulting matrix merges the above, and collapses columns to reslt in a median m/z, and median retention time, and then the intensity of each pool, and NA where results are missing.](../../images/lcmspreproc_slides_3.1.png)
+![Large set of tables, in three groups, with each time 3 tables representing 3 pools. The top group is labelled "independent peaklists". The next row of tables is labelled "group ions by m/z", and the rows in the tables are coloured based on m/z values and lined up by their colours. The third group is labelled "group by retention time", and the coloured rows are now merged into one big table, each colour on its own row, with this time some blue in two shades (same m/z but different rt). The resulting matrix merges the above, and collapses columns to result in a median m/z, and median retention time, and then the intensity of each pool; NA are filled when the line was missing for some pools.](../../images/lcmspreproc_slides_3.1.png)
 
 ---
 
 ## xcms - groupChromPeaks - Alignment group
 
-![Another text blob and image combo. The text reads: first, a binning of mass domain is performed. The size of the bin defined by the width of overlapping m/z slices. THen for each m/z bin, all ions of all samples are taken into account on the whole. a kernel density estimator is used to detect regions of retention time with high density ions. Below the graph shows many individual peaks, clustered.](../../images/lcmspreproc_slides_3.2.png)
+![Another text blob and image combo. The text: "First, a binning of mass domain is performed. The size of the bin defined by the width of overlapping m/z slices. Then for each m/z bin, all ions of all samples are taken into account on the whole. A kernel density estimator is used to detect regions of retention time with high density ions." Below the graph shows an example of plot output from groupChromPeaks, where is represented, for one m/z slice, peaks found in samples as points along the retention time.](../../images/lcmspreproc_slides_3.2.png)
```

Comment:
plot output from groupChromPeaks, which represents peaks found in samples as points along the retention time for one m/z slice